pkg/api/constants.go

```diff
@@ -81,10 +81,10 @@ const (
 
 	// Scheduler Inspect API: API to inspect current scheduling status
 	// Notes:
-	// 1. Both Binding and Bound AffinityGroups/Pods are considered as Allocated.
+	// 1. Both Binding and Bound PodGroups/Pods are considered as Allocated.
 	InspectPath = VersionPath + "/inspect"
-	// Inspect current allocated AffinityGroup(s)
-	AffinityGroupsPath = InspectPath + "/affinitygroups/"
+	// Inspect current allocated PodGroup(s)
+	PodGroupsPath = InspectPath + "/podgroups/"
 	// Inspect current cluster status
```

Comment:
May need restserver change?

Reply:
yes, need changes in https://github.com/microsoft/pai/pull/5428, but rest server will still work if cannot get data from old endpoint